migrations/1641484605043_make-items-urn-suffix-unique.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
is this necessary?

Reply:
It's not! It was put in place by the node-pg-migrate template.